src/corrplot.jl

```diff
@@ -65,7 +69,7 @@ end
                 #above diag... hist2d
                 @series begin
                     seriestype := get(plotattributes, :seriestype, :histogram2d)
-                    if title != "" && i == 1 && j == div(n,2)+1
+                    if title != "" && i == 1 && ((title_location == :center && j == div(n,2)+1) || (title_location == :right && j == n))
```

Comment:
Maybe set `title_location := :left` if `n` is an even number? It would move it further towards the center, not sure it would really look nicer though.

Reply:
That would probably look nicer. Thanks for the hint!